truenas_api_client/__init__.py

```diff
@@ -409,28 +674,39 @@ def on_close(self, code, reason=None):
                 job['exc_info'] = {
                     'type': 'Exception',
                     'repr': error,
-                    'extra': None,
+                    'extra': [],
```

Comment:
Why?

Reply:
Another type checker change that ensures `extra` is always a `list`. Ultimately this is because the argument to `ValidationErrors.__init__()` cannot be `None`.

Reply:
`extra` is `None` by default for `CallError`.